diff --git a/src/core/ngx_log.c b/src/core/ngx_log.c
index 770a590..26dd221 100644
--- a/src/core/ngx_log.c
+++ b/src/core/ngx_log.c
@@ -66,7 +66,7 @@ static ngx_str_t err_levels[] = {
 
 static const char *debug_levels[] = {
     "debug_core", "debug_alloc", "debug_mutex", "debug_event",
-    "debug_http", "debug_mail", "debug_mysql"
+    "debug_http", "debug_mail", "debug_mysql", "debug_tcp"
 };
 
 
diff --git a/src/core/ngx_log.h b/src/core/ngx_log.h
index b736aa2..0b1507b 100644
--- a/src/core/ngx_log.h
+++ b/src/core/ngx_log.h
@@ -29,6 +29,7 @@
 #define NGX_LOG_DEBUG_HTTP        0x100
 #define NGX_LOG_DEBUG_MAIL        0x200
 #define NGX_LOG_DEBUG_MYSQL       0x400
+#define NGX_LOG_DEBUG_TCP         0x800
 
 /*
  * do not forget to update debug_levels[] in src/core/ngx_log.c
@@ -36,7 +37,7 @@
  */
 
 #define NGX_LOG_DEBUG_FIRST       NGX_LOG_DEBUG_CORE
-#define NGX_LOG_DEBUG_LAST        NGX_LOG_DEBUG_MYSQL
+#define NGX_LOG_DEBUG_LAST        NGX_LOG_DEBUG_TCP
 #define NGX_LOG_DEBUG_CONNECTION  0x80000000
 #define NGX_LOG_DEBUG_ALL         0x7ffffff0
 
